utils/nctl/sh/scenarios/common/itst.sh

```diff
@@ -129,6 +129,35 @@ function get_reactor_state() {
     echo "$REACTOR_STATE"
 }
 
+function check_nodes_1_to_5_sync_fast() {
```

Comment:
nit
```suggestion
function parallel_check_nodes_1_to_5_sync() {
```

Reply:
Renamed in https://github.com/casper-network/casper-node/pull/3936/commits/d2b58bb7e0e667cfc8ee682ba8bdfe8fadd36e8c